pkg/controller/elasticsearch/validation/webhook.go

```diff
@@ -55,13 +59,24 @@ func (wh *validatingWebhook) InjectDecoder(d *admission.Decoder) error {
 }
 
 func (wh *validatingWebhook) validateCreate(es esv1.Elasticsearch) error {
+	// If this Elasticsearch instance is not within the set of managed namespaces
+	// for this operator ignore this request.
+	if !slices.Contains(wh.managedNamespaces, es.Namespace) {
```

Comment:
We should also check that `len(wh.managedNamespaces) > 0` , otherwise the validation is skipped if the operator is watching all the namespaces.